### DISCUSSION TITLE
What’s the nuget package repo link for GH Enterprise

### DISCUSSION THREAD

**Discussion by e-master** [community/community#26555] [Enterprise]
<p>We’ve got GitHub Enterprise 3.0.1, and our admin turned on the nuget package support, however I cannot figure out the url to upload packages to. I tried <a href="https://nuget.pkg.ourgithub.mycompany.com/mylogin/myrepo/index.json" rel="noopener nofollow ugc">https://nuget.pkg.ourgithub.mycompany.com/mylogin/myrepo/index.json</a>, but curl kept complaining that it can’t resolve the domain name. I have the feeling that our proxy is not configured to accept sub domains, but at the same time the official docs don’t show what exact path should I use in an on premise github server for Packages, so I also suspect I’m using the wrong url.</p>

**Comment by jcansdale** [comment, marked as answer]
<p>Hi <a class="mention" href="/e-master">@e-master</a>,</p>
<blockquote>
<p>I have the feeling that our proxy is not configured to accept sub domains, but at the same time the official docs don’t show what exact path should I use in an on premise github server for Packages, so I also suspect I’m using the wrong url.</p>
</blockquote>
<p>If your instance has subdomain isolation disabled, you should be able to use the following url:</p>
<pre><code class="lang-auto">https://HOSTNAME/_registry/nuget/OWNER/index.json
</code></pre>
<p>See <a href="https://docs.github.com/en/enterprise-server@2.22/packages/guides/configuring-dotnet-cli-for-use-with-github-packages#authenticating-with-a-personal-access-token:~:text=If%20your%20instance%20has%20subdomain%20isolation%20disabled" rel="noopener nofollow ugc">here</a> for more information.</p>
<p>I hope that helps!</p>

**Comment by ghost** [comment]
<aside class="quote group-github-employees" data-username="jcansdale" data-post="2" data-topic="167172">
<div class="title">
<div class="quote-controls"></div>
 jcansdale:</div>
<blockquote>
<p><code>https://HOSTNAME/_registry/nuget/OWNER/index.json</code></p>
</blockquote>
</aside>
<p>It doesn’t work for me. when I replace hostname with my GH Enterprise URL and OWNER with my organization name or username, it didn’t find index.json. Is there any configuration on GH Enterprise missing? Are the Packages registry are enabled on the Managment Console.</p>

**Comment by jcansdale** [comment]
<p><a class="mention" href="/douglasrdc">@douglasrdc</a>,</p>
<aside class="quote no-group" data-username="douglasrdc" data-post="3" data-topic="167172">
<div class="title">
<div class="quote-controls"></div>
 douglasrdc:</div>
<blockquote>
<p>when I replace hostname with my GH Enterprise URL and OWNER with my organization name or username, it didn’t find index.json.</p>
</blockquote>
</aside>
<p>Can I check that your server has subdomain isolation disabled? If it doesn’t, the feed URL would look like this <code>https://nuget.HOSTNAME/OWNER/index.json</code>.</p>

**Comment by ghost** [comment]
<p>Hi <a class="mention" href="/jcansdale">@jcansdale</a> , Thank you for your help. I got it working now. have to set up a subdomain like you said: <a href="https://nuget.mydomain/organization/index.json" rel="noopener nofollow ugc">https://nuget.mydomain/organization/index.json</a> and set up SSL on this URL. Now it is working fine.</p>
